e2e/definitions/subject/alpine.yaml

```
@@ -0,0 +1,26 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: subject-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: subject
+  template:
+    metadata:
+      labels:
+        app: subject
+    spec:
+      containers:
+      - name: subject-container
+        image: alpine
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - name: volume
+          mountPath: /pvc/
```

I propose a real app. Besides, `alpine` again pulls from Docker Hub, which I would avoid.
Redis has reasonably low image size (40mb) and fast container startup (1s)
```suggestion
      - name: subject-container
        image: quay.io/bitnami/redis:latest
        imagePullPolicy: IfNotPresent
        env:
        - name: REDIS_PASSWORD
          value: e2e-test
        volumeMounts:
        - name: volume
          mountPath: /bitnami/redis
```